matcher.py

```diff
@@ -52,7 +64,7 @@ def removeStopwords(ls):
     
     # CONSTANTS
     BLEU_THRESHOLD = 0.4
-    LEXICAL_THRESHOLD = 0.5 # Note: changing this value didn't change the ordering of the tested systems
+    LEXICAL_THRESHOLD = 0.25 # Note: changing this value didn't change the ordering of the tested systems
```

Comment:
This threshold seems to have been this high as it was overcompensating for this matching bug. i.e. at 50% it was able to better increase the likelihood and alignment of comparing the gold extraction with the "right" predicted extraction. Now that we ensure that it is comparing two extractions with the same predicate in the matching function, it seems like this can now be lowered to better support packages that might not include as many modifiers/adjectives in the subject and object arguments, especially with small extractions with only a few words.  It also helps in the case of conjunction splitting (for packages that support that).